content/docs/overview.md

```diff
@@ -1802,6 +1825,18 @@ x := p^ // ^ on the right
 
 **Note:** Unlike C, Odin has no pointer arithmetic. If you need a form of pointer arithmetic, please use the `ptr_offset` and `ptr_sub` procedures in the `"core:mem"` package.
 
+**Note:** Unlike function pointers in C, procedures are ordinary values, so you don't need a function pointer to have a mutable proc variable that can contain different procedures.
+Since a proc is an ordinary value, a pointer to a proc can be used to mutate a proc variable, and change what it is pointing to:
+```odin
+change_proc_ptr :: proc(p : ^proc(int)->int) {
+	p^ = proc(x:int) -> int { return x + 1 }
+}
```

Comment:
This is probably more confusing because people will scan this and think you are meant to pass a procedure by pointer, even if it is clear from context you shouldn't.

Reply:
Yeah, agreed. 🤔 maybe a better example would be to have a procedure that returns one procedure or another based on parameter, and assign `p` to each of those at the call site?
Something like
```
select_proc :: proc(s : int) -> proc(int)->int {
	switch {
	case n == 0:
		return proc(x:int) -> int { return x }
	case n == 1:
		return proc(x:int) -> int { return x + 1 }
	}
}
p := select_proc(0)
fmt.println(p(1))  // 1
p := select_proc(1)
fmt.println(p(1))  // 2
```
?